source/slang/slang-diagnostic-defs.h

```diff
@@ -482,6 +483,12 @@ DIAGNOSTIC(40007, Internal, irValidationFailed, "IR validation failed: $0")
 
 DIAGNOSTIC(40008, Error, invalidLValueForRefParameter, "the form of this l-value argument is not valid for a `ref` parameter")
 
+DIAGNOSTIC(40009, Error, dynamicInterfaceLacksAnyValueSizeAttribute, "interface '$0' is being used in dynamic dispatch code but has no [anyValueSize] attribute defined.")
+DIAGNOSTIC(40010, Note, seeInterfaceUsage, "see usage of interface '$0'.")
+
+DIAGNOSTIC(40011, Error, unconstrainedGenericParameterNotAllowedInDynamicFunction, "unconstrained generic paramter '$0' is not allowed in a dynamic function.")
```

Comment:
I assume this error could be removed if/when we allow `[anyValueSize(...)]` to be applied to a generic type parameter.